### DISCUSSION TITLE
Large PR shows infinite Files changed in tab and zero files in diff

### DISCUSSION THREAD

**Discussion by rdhelms** [community/community#35101] [Pull Requests]
I was under the impression that even for large PRs, GitHub would display the correct number of changed files and that it would just be within each file's expandable section that there might be a "Too large to display" message. Is it the intended behavior that some large PRs will just be completely unable to show the correct number of changed files? Can we not at least see the diffs for the files with smaller changes?

<img width="1021" alt="image" src="https://user-images.githubusercontent.com/7010586/193877160-3df54cb4-d46c-455a-9db5-2b691ebb2746.png">

**Comment by sheeeng** [comment]
Any updates from GitHub on this issue?
